app/scripts/controllers/mmi-controller.js

```diff
@@ -39,7 +38,12 @@ export default class MMIController extends EventEmitter {
     this.signatureController = opts.signatureController;
     this.platform = opts.platform;
     this.extension = opts.extension;
-    this.trackTransactionEvents = opts.trackTransactionEvents;
+
+    const { hooks } = opts ?? {};
```

Comment:
I'm not sure I'd call these hooks since they're not changing the flow of the MMI logic, but rather just providing the required data to function.

Would it add clarity to leave these as standard constructor options?